src/librbd/deep_copy/ImageCopyRequest.cc

```diff
@@ -147,11 +147,8 @@ void ImageCopyRequest<I>::send_object_copies() {
 
     // attempt to schedule at least 'max_ops' initial requests where
     // some objects might be skipped if fast-diff notes no change
```

Comment:
I think this comment is not entirely correct now. May be just remove it?

Reply:
I decided to keep it.  Technically, it is correct, even though the meaning is a bit different now:
- "attempt to schedule at least max_ops initial requests" is there: if the image has just a couple of objects or the deep-copy request is canceled too quickly, fewer than max_ops requests are scheduled (as before)
- "some objects might be skipped if fast-diff notes no change" is also there: `ObjectCopyRequest` instances are not created for such objects (as before) but the callback is arranged to be called (new behavior)